changelog/next/features/3461-fluent-bit.md

```diff
@@ -0,0 +1,3 @@
+The new `fluent-bit` source and sink operator provide and interface to the
```

Comment:
It's not a connector, though. Otherwise I'd be able to use `from fluent-bit`. It is its own operator because Fluent Bit has an internal structured format.

Reply:
Yeaah, you're right.
I guess the phrase "source and sink operator" made me suspicious.